### PR TITLE
feat: Added notebook demonstrating how to evaluate a forecasting training job.

### PR DESCRIPTION
**Pull request by mansiachuthan**
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/master/notebooks/official) folder, follow this mandatory checklist:
- [ ] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/notebooks/notebook_template.ipynb) as a starting point.
- [ ] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [ ] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/master/notebooks/community) folder:
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/docs/CODEOWNERS) file under the `# Community Notebooks` section, pointing to the author or the author's team.

If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/master/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/docs/CODEOWNERS) file under the `# Community Content` section, pointing to the author or the author's team.

### CODE REVIEW

**Comment by mansiachuthan** [comment]
No CODEOWNERS file for the directory I contributed to (notebooks/community/sdk)

File: notebooks/community/sdk/sdk_automl_forecasting_evaluating_a_model.ipynb
Comment (ivanmkc) [inline notebooks/community/sdk/sdk_automl_forecasting_evaluating_a_model.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #8.        column_transformations=[
</pre><p>Use column_specs instead of column_transformations (deprecated)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

**Review by ivanmkc — APPROVED** [review]
LGTM pending comment

File: notebooks/community/sdk/sdk_automl_forecasting_evaluating_a_model.ipynb
Comment (thehardikv) [reply notebooks/community/sdk/sdk_automl_forecasting_evaluating_a_model.ipynb:1]: Done.

**Comment by ivanmkc** [comment]
CODEOWNERS file change required. 

See https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/notebooks/community/CODEOWNERS

**Pull request merged**
